inst/@sym/sort.m

```diff
@@ -0,0 +1,82 @@
+%% Copyright (C) 2014, 2015 Utkarsh Gautam
+%%
+%% This file is part of OctSymPy.
+%%
+%% OctSymPy is free software; you can redistribute it and/or modify
+%% it under the terms of the GNU General Public License as published
+%% by the Free Software Foundation; either version 3 of the License,
+%% or (at your option) any later version.
+%%
+%% This software is distributed in the hope that it will be useful,
+%% but WITHOUT ANY WARRANTY; without even the implied warranty
+%% of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See
+%% the GNU General Public License for more details.
+%%
+%% You should have received a copy of the GNU General Public
+%% License along with this software; see the file COPYING.
+%% If not, see <http://www.gnu.org/licenses/>.
+
+%% -*- texinfo -*-
+%% @documentencoding UTF-8
+%% @deftypefn  {Function File} {@var{g} =} sort (@var{f})
+%% Sorting symbols .
```

Comment:
extra space before full stop and maybe not the best one-liner: it actually can only sort Numbers.

Maybe (read this)[https://www.gnu.org/software/octave/doc/interpreter/Documentation-Tips.html]